tools/license2rtf.js

```diff
@@ -1,15 +1,16 @@
+'use strict';
 
 var assert = require('assert'),
-    Stream = require('stream'),
-    inherits = require('util').inherits;
+  Stream = require('stream'),
+  inherits = require('util').inherits;
```

Comment:
`one-var: [2, {uninitialized: never}]` gives 26 errors, 4 in `lib` which looks acceptable. Also doing it for initialized variables gives over 100 though.